e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -47,10 +47,10 @@
                 "value": "true"
               },
               "experimentalfeatures__enabled": {
-                  "value": "true"
+                "value": "true"
```

Comment:
thanks for clean up.

Reply:
don't thank me, thank my IDE who did it automatically